### PR TITLE
Adding fec docs

### PR DESCRIPTION
**Pull request by arivepr**
Adding main FEC documentation to docs page.

### CODE REVIEW

File: packages/docs/pages/fec/index.md [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+module.exports = {
+    presets: [
+        // list of your presets goes here
+    ],
+    plugins: [
+        // your other plugins
+        [
+            'transform-imports',
+            {
+              '@patternfly/react-core': {
```
Comment (Hyperkid123) [inline packages/docs/pages/fec/index.md:55]: Can you delete the react-core example. We only use the one for icons.

File: packages/docs/pages/fec/index.md [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+
+module.exports = {
+    presets: [
+        // list of your presets goes here
+    ],
+    plugins: [
+        // your other plugins
+        [
+            'transform-imports',
+<<<<<<< HEAD
```
Comment (Hyperkid123) [inline packages/docs/pages/fec/index.md:54]: You have some unresolved merge conflicts

**Review by Hyperkid123 — CHANGES_REQUESTED** [review]
@arivepr you till have git merge conflict characters in the [packages/docs/pages/fec/index.md](https://github.com/RedHatInsights/frontend-components/pull/1504/files#diff-a90130f5fab033de2dedf63f2a9fbe0b1b60d0e3a12f666367bbb5da819c8334) file .

**Comment by arivepr** [comment]
> @arivepr you till have git merge conflict characters in the [packages/docs/pages/fec/index.md](https://github.com/RedHatInsights/frontend-components/pull/1504/files#diff-a90130f5fab033de2dedf63f2a9fbe0b1b60d0e3a12f666367bbb5da819c8334) file .

Don't know what happened there, should be good now

File: packages/docs/pages/fec/index.md [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+  TextListVariants: 'TextList'
+};
+
+module.exports = {
+    presets: [
+        // list of your presets goes here
+    ],
+    plugins: [
+        // your other plugins
+        [
```
Comment (Hyperkid123) [inline packages/docs/pages/fec/index.md:52]: This array entry should not exist. Only the one for icons :axe:

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**